hivemind/dht/node.py

```diff
@@ -235,7 +235,11 @@ async def create(
                 finished_pings |= finished_in_time
 
             if not finished_pings or all(ping.result() is None for ping in finished_pings):
-                logger.warning("DHTNode bootstrap failed: none of the initial_peers responded to a ping.")
+                message = "DHTNode bootstrap failed: none of the initial_peers responded to a ping."
+                if validate:
+                    raise RuntimeError(f"{message} (set validate=False to ignore)")
+                else:
+                    logger.warning(message)
```

Comment:
```suggestion
                raise RuntimeError("DHTNode bootstrap failed: none of the initial_peers responded to a ping")
```

Reply:
Summary of a short debate: while there are indeed reasons to always throw this warning, we ultimately agreed to keep the old behavior for 2 reasons:
* validate is True by default and it is difficult to change it without being very explicit
* there is a use case there not raising error is preferable: when several peers use pre-generated peer ids, each peer should connect to others, but some peer will inevitably become "first". Thus, we will need to set this flag to avoid try-except clauses and repeated daemon creation.

However, we also agreed to rename this flag to something more specific.